### DISCUSSION TITLE
如何在 Objective-C 中实现协议扩展 · /protocol-extension

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#278] [Announcements]
https://draveness.me/protocol-extension

**Comment by tangzzz-fan** [comment]
你好, 我在使用过程中, 没有使用控制台这种文件都放在一起的形式, 一个 model 对象遵守<TestProtocol> 协议之后, 在使用 model 对象的地方, 只要一导入这个 model 头文件就说有重复标识(还没有使用 model 创建对象). 看了源码, 没有发现可以解释的地方, 想咨询一下. 谢谢.
